api/write.go

```diff
@@ -206,9 +206,11 @@ func (w *WriteAPIImpl) WritePoint(point *write.Point) {
 	line, err := w.service.EncodePoints(point)
 	if err != nil {
 		log.Errorf("point encoding error: %s\n", err.Error())
-	} else {
-		w.bufferCh <- line
+		if w.errCh != nil {
+			w.errCh <- err	
+		}
 	}
+	w.bufferCh <- line
```

Comment:
`line` is empty in case of an error. It should be on the other branch of if

Reply:
Ah good catch. Updated. Thanks.